src/ConfigR/BasicConfig.Dictionary.cs

```diff
@@ -57,7 +59,7 @@ public bool ContainsKey(string key)
 
         public bool Remove(string key)
         {
-            log.TraceFormat(CultureInfo.InvariantCulture, "Removing '{0}' from {1}", key, this.GetSource());
+            log.TraceFormat(string.Format(CultureInfo.InvariantCulture, "Removing '{0}' from {1}", key, this.GetSource()));

```

Comment:
Just string. I would prefer Log.Trace( () => string); for lazy loading. Thoughts?

CM

-----Original Message-----
From: "Adam Ralph" <notifications@github.com>
Sent: ‎08/‎10/‎2015 18:08
To: "config-r/config-r" <config-r@noreply.github.com>
Cc: "Christos Matskas" <christos.matskas@softwarelounge.co.uk>
Subject: Re: [config-r] Logging uprade to LibLog (#205)

In src/ConfigR/BasicConfig.Dictionary.cs:
> @@ -57,7 +59,7 @@ public bool ContainsKey(string key)
>  
>          public bool Remove(string key)
>          {
> -            log.TraceFormat(CultureInfo.InvariantCulture, "Removing '{0}' from {1}", key, this.GetSource());
> +            log.TraceFormat(string.Format(CultureInfo.InvariantCulture, "Removing '{0}' from {1}", key, this.GetSource()));
Does the LibLog TraceFormat not accept formatting parameters?
—
Reply to this email directly or view it on GitHub.